styles/styles.css

```diff
@@ -15,6 +15,32 @@ header {
   box-shadow: 15px -15px #be7f6a;
 }
 
+.card1,
+.question,
+.answer {
+  background-color: #87adcf;
+  text-align: center;
```

Comment:
These properties are inherited, so you can only pass it to `card`.

Reply:
tried to delete .question, but it didn't look good; background of question turned #bfbfbd instead of staying blue.

Reply:
> tried to delete .question, but it didn't look good; background of question turned #bfbfbd instead of staying blue.

this is a comment I originally answered on your "Do you really want to pass the same values to card, question, and answer?" .. something went wrong here when I pushed.